service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java

```diff
@@ -491,6 +492,46 @@ void shouldNotHaveCaseIdPropertyWhenRoboticsDataDeserializes() throws IOExceptio
         }
     }
 
+    @Nested
+    class FeePaid {
+        private CaseData.CaseDataBuilder caseDataBuilder;
+
+        @BeforeEach
+        void setup() {
+            caseDataBuilder = prepareCaseData().toBuilder();
+        }
+
+        @Test
+        void shouldGetFeePaidFromCaseData() throws JsonProcessingException {
+            caseDataBuilder.amountToPay("100000");
+
+            RoboticsData roboticsData = roboticsDataService.prepareRoboticsData(caseDataBuilder.build(), CASE_ID);
+            String returnedRoboticsJson = roboticsDataService.convertRoboticsDataToJson(roboticsData);
+
+            Map<String, Object> returnedRoboticsDataMap = objectMapper.reader()
+                .forType(new TypeReference<Map<String, Object>>() {
+                })
+                .readValue(returnedRoboticsJson);
```

Comment:
Can be simplified to `mapper.readValue(returnedRoboticsJson, new TypeReference<>())` I believe

Reply:
same for test below as well

Reply:
done